### ISSUE TITLE
 License:  UNKNOWN ... Free To Use But Restricted

### ISSUE THREAD

**Issue by APolyakova** [atoti/atoti#279]
Question posted on request by Aurelia Mason.

When I tried atoti, I immediately hit a snag. Step 1 involves onboarding the open-source component onto our artifactory (private PyPI).
As part of this, the OSS software onboarding service runs various license / security checks. The atoti library was rejected because of license issues related to its dependencies. Are these things your team can have fixed e.g. by submitting a PR for jdk4py repo or alternatively, finding an alternative library if one exists ?

```
RejectReasons (1)
   RejectReason:         0a9090c5-32eb-4100-8917-e441447c0d1c
     Type:     UNKNOWN_LICENSE_FOUND
     License:  UNKNOWN ... Free To Use But Restricted
 Dependencies (10)
   Dependency:          PYPI - PyYAML==5.4.1
   Dependency:          PYPI - jdk4py==11.0.9.0
     RejectReasons (1)
       RejectReason:     b9dade02-031d-4a0a-a488-48e7d0ca3c91
         Type:     UNKNOWN_LICENSE_FOUND
         License:  UNKNOWN
```

**Comment by rcolle** [comment]
Hi Anastasia / Aurelia,

jdk4py is licensed under the GPLv2 license, one of the most commonly used OSS software. 
See https://github.com/atoti/jdk4py/blob/master/LICENSE

Can we have more details on the actual issue?

Thanks

**Comment by tibdex** [comment]
Maybe we lack a `License ::` classifier here: https://github.com/atoti/jdk4py/blob/8810e2faad3b7eb9fee980a99ce1ae686ea6e2a4/setup.py#L52-L56.

**Comment by tibdex** [comment]
Fixed in atoti/jdk4py#33.
